__tests__/database/schema/UserSchema.test.ts

```diff
@@ -470,7 +470,8 @@ describe('TESTING UserSchema', () => {
             name: 'username',
             last_active: 'lastActiveAt',
             created_at: 'createdAt',
-            image: 'profilePicture'
+            image: 'profilePicture',
+            username: 'username'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There seems to be a duplicate key `username` in the object. This might lead to unexpected behavior as the second `username` will overwrite the first one.
```diff
             name: 'username',
             last_active: 'lastActiveAt',
             created_at: 'createdAt',
-            image: 'profilePicture',
-            username: 'username'
+            image: 'profilePicture'
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->